### PR TITLE
Repair devsite docs

### PR DESCRIPTION
**Pull request by dlarocque**
I ran `yarn docgen:all` and noticed there were some changes. These should have been caught by our docgen check job, I'm not sure why they weren't.

### CODE REVIEW

File: docs-devsite/_toc.yaml
Comment (egilmorez) [inline docs-devsite/_toc.yaml:1]: This isn't a new file -- shouldn't there be a diff, probably just adding the new vector value files?

**Review by egilmorez — COMMENTED** [review]
Thanks!

One question about the TOC.

File: docs-devsite/_toc.yaml
Comment (dlarocque) [reply docs-devsite/_toc.yaml:1]: Git thinks it's a new file because the TOC was accidentally removed in #8215.

**Review by egilmorez — APPROVED** [review]
LGTM, thanks for noting and fixing!

**Pull request merged**